client/controller/models/apps/apps.go

```diff
@@ -92,7 +92,7 @@ func Logs(c *client.Client, appID string, lines int) (string, error) {
 	u := fmt.Sprintf("/v2/apps/%s/logs", appID)
 
 	if lines > 0 {
-		u += "?log_lines=" + strconv.Itoa(lines)
+		u += "/?log_lines=" + strconv.Itoa(lines)
```

Comment:
Do logger URLs absolutely need a slash at the end now?  It seems a generally accepted practice that the final trailing slash in a URL is irrelevant.